cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -133,14 +138,22 @@
                 </span>
                 {% endif %}
                 <div class="{{ _classes( nav_depth, '-lists' ) }}">
-                    {{ _nav_list( nav_depth, nav_items ) }}
+                    {{ _nav_list( nav_depth, nav_groups ) }}
                 </div>
+               {% if footer %}
+          <div class="nav_footer">
+            {% for item in footer %}
+            <p>{{ item | safe }}</p>
+            {% endfor %}
+          </div>
+         {% endif %}
```

Comment:
Trivial: Please adjust indentation so the `div` tags align with the div tags above, and use four-space indentation.